kubernetes/Packing.cmake

```diff
@@ -0,0 +1,22 @@
+set(CPACK_PACKAGE_NAME lib${pkgName})
+
+set(CPACK_PACKAGE_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
+
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY ${PROJECT_PACKAGE_DESCRIPTION_SUMMARY})
+set(CPACK_PACKAGE_VENDOR ${PROJECT_PACKAGE_VENDOR})
+set(CPACK_PACKAGE_CONTACT ${PROJECT_PACKAGE_CONTACT})
+set(CPACK_DEBIAN_PACKAGE_MAINTAINER ${PROJECT_PACKAGE_MAINTAINER})
+
+set(CPACK_VERBATIM_VARIABLES YES)
```

Comment:
Enable adding required dependencies' info using the following variable:
`set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS TRUE)`
Note that on my machine this only added OpenSSL. You might consider adding libyaml and libwebsockets manually using `CPACK_DEBIAN_PACKAGE_DEPENDS`.

Reply:
Thank you @ahmedyarub 

`set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS TRUE)` is good, in my build env, it generates the depenencies:
```
Depends: libc6 (>= 2.17), libcurl4 (>= 7.56.1), libssl1.1 (>= 1.1.0)
```

Currently (before Ubuntu 22.04 LTS releases), `libwebsockets` must be installed from source code for the C client. So `CPACK_DEBIAN_PACKAGE_DEPENDS` is not needed now.